src/compas/datastructures/mesh/operations/substitute.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+
+__all__ = [
+	'mesh_substitute_vertex_in_faces'
+]
+
+
+def mesh_substitute_vertex_in_faces(mesh, old_vkey, new_vkey, fkeys=None):
+	"""Substitute in a mesh a vertex by another one.
+	In all faces by default or in a given set of faces.
+
+	Parameters
+	----------
+	old_vkey : hashable
+		The old vertex key.
+	new_vkey : hashable
+		The new vertex key.
+	fkeys : list, optional
+		List of face keys where to subsitute the old vertex by the new one.
+		Default is to subsitute in all faces.
+
+	"""
```

Comment:
should the function not return the new fkeys?

Reply:
added as output